src/tests/contract/data/person-identity-items.json

```diff
@@ -0,0 +1,99 @@
+{
+    "sessionId": {
```

Comment:
I have added the code to populate the PI data, but i dont see it being used as we are not writing tests for session endpoint.